src/controllers/pages/student.js

```diff
@@ -13,11 +13,23 @@ export default class StudentController{
     }
 
     static cancelCheckout() {
-        Dispatcher.handleAction('CLEAR_ITEMS');
+        Dispatcher.handleAction('CLEAR_CONTENTS');
```

Comment:
Many things contain contents. Therein lies the ambiguity.

Should this action clear the contents of the error store? Should it clear the contents of a student's inventory?